### PR TITLE
#6367 (#6397 step 1) migration

### PR DESCRIPTION
**Pull request by juanignaciosl**
This is the migration of the PR #6397. @gfiorav CR this, please.

### CODE REVIEW

File: db/migrate/20160114082200_add_created_via_to_user_creations.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Sequel.migration do
+  up do
+    add_column :user_creations, :created_via, :text, null: false, default: 'org_signup'
+    Rails::Sequel.connection.run("update user_creations set created_via = 'api' where created_via_api = 'true'")
+  end
```
Comment (gfiorav) [inline db/migrate/20160114082200_add_created_via_to_user_creations.rb]: How about deleting the `created_via_api` column after this?

Comment (juanignaciosl) [inline db/migrate/20160114082200_add_created_via_to_user_creations.rb]: Sure, but when you drop a column in a database in zero-downtime, balanced server environment (such as ours) you must "disable" it first. You'll see [the disabling in the source PR of this migration](https://github.com/CartoDB/cartodb/pull/6397/files#diff-4a31751081269847a70e8278080e8e89R5).

You can find the `drop_column` at the main PR (#6378) I'm splitting into 3 steps (5 deploys, I think). My plan is in this comment: https://github.com/CartoDB/cartodb/pull/6378#issuecomment-171373546

This is a great article about it, please read it, it has many interesting details: http://blog.codeship.com/rails-migrations-zero-downtime/

Comment (gfiorav) [inline db/migrate/20160114082200_add_created_via_to_user_creations.rb]: :ok: 

Didn't know about your plan!

**Comment by gfiorav** [comment]
:+1:

File: db/migrate/20160114082200_add_created_via_to_user_creations.rb (hunk 1 above)
Comment (juanignaciosl) [inline db/migrate/20160114082200_add_created_via_to_user_creations.rb]: Yep, I should've given you more context :-)

**Pull request merged**